test/test_va_api_createbuffer.cpp

```diff
@@ -168,7 +170,8 @@ TEST_P(VAAPICreateBuffer, CreateBufferWithOutData)
             // vaCreateContext input requires resolution, since this test
             // doesn't create surfaces, passing 1x1 resolution should provide
             // the desired result.
-            doCreateContext(Resolution(1, 1));
+            doGetMinSurfaceResolution(currentTestInput.inputConfig.first, currentTestInput.inputConfig.second, minResolution);
```

Comment:
Please wrap this line.

Reply:
@uartie ,i am not sure what is the idea of wrap? would you explain it?